runtime.md

```diff
@@ -35,6 +35,7 @@ The lifecycle describes the timeline of events that happen from when a container
 1. OCI compliant runtime's `create` command is invoked with a reference to the location of the bundle and a unique identifier.
    How these references are passed to the runtime is an implementation detail.
 2. The container's runtime environment MUST be created according to the configuration in [`config.json`](config.md).
+   If the runtime is unable to create the runtimme environment specified in the [`config.json`](config.md) it MUST generate an error.
```

Comment:
Also, you probably want a comma after `config.json`: “If … `config.json`, it MUST generate an error.”